store/src/main/java/com/onlineStore/coherent/multithreading/ThreadDeleteOrder.java

```diff
@@ -1,29 +1,24 @@
 package com.onlineStore.coherent.multithreading;
 
-
-import database.DataBase;
-import product.Product;
+import com.onlineShop.coherent.product.Product;
 
 import java.util.List;
 
 public class ThreadDeleteOrder extends Thread {
```

Comment:
The method should be updated to clean order table.
private List<Product> order should be removed from the method

Reply:
private List<Product> order should be removed